.gitignore

```diff
@@ -2,3 +2,6 @@ target/
 **/*.rs.bk
 Cargo.lock
 tags
+
+# intellij IDE
+.idea
```

Comment:
Hope this is ok.

Reply:
You probably want this in your [global .gitignore](https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

Reply:
Gotchya, will get that out later today.

Reply:
fixed